src/site/filters/liquid.js

```diff
@@ -12,6 +12,10 @@ function getPath(obj) {
 module.exports = function registerFilters() {
   const { cmsFeatureFlags } = global;
 
+  // Set timeout option to something higher (20mins)
+  // eslint-disable-next-line no-new
+  new liquid.Context({ timeout: 1200000 });
```

Comment:
Curious to know why this was needed.

Reply:
@mpelzsherman as the node count increases the parsing time for templates also goes up. The TinyLiquid plugin we're using has a 2min timeout, so I increased it

Reply:
OK. I don't know enough about how we're using TinyLiquid to understand why we are hitting this limit. Is it timing out while rendering a single page, or does the timeout need to cover the total time to render all pages?

Reply:
It only happened with the node increase to 12k, so very likely the time to render all pages unless there's a single gigantic page buried in there somewhere